presets/4.3/tune/mouseFPV_Digipick_braced.txt

```diff
@@ -0,0 +1,299 @@
+
+# -- Yaw lowpass --
+set yaw_lowpass_hz = 100
+
+
+
+# ------ OPTIONS GO BELOW THIS LINE ------
+#$ OPTION_GROUP BEGIN: Filters (Choose One or None)
+
+    #$ OPTION BEGIN (CHECKED): RPM Filters DShot600 (F7 & Up)
+        # -- End Defaults --
+        # -- Begin Mouse Tune --
+
+        # enable dshot rpm telemetry
+        set motor_pwm_protocol = DSHOT600
+        set dshot_bidir = ON
+        set motor_poles = 12
+
+        # -- Gyro lowpass filters --
+        # -- No Gyro Lowpass
+        set gyro_lpf1_static_hz = 0
+        set gyro_lpf1_dyn_min_hz = 0
+        set gyro_lpf1_dyn_max_hz = 0
+        set simplified_gyro_filter = on
+        set simplified_gyro_filter_multiplier = 150
+        simplified_tuning apply
+
+        # -- Gyro Dynamic Notches --
+        set dyn_notch_count = 1
+        set dyn_notch_q = 500
+        set dyn_notch_min_hz = 100
+        set dyn_notch_max_hz = 550
+
+        # -- RPM filtering --
+        set rpm_filter_q = 500
+        set rpm_filter_min_hz = 100
+        set rpm_filter_fade_range_hz = 60
+        set rpm_filter_harmonics = 3
+
+        # -- Dterm filtering --
+        set simplified_dterm_filter = on
+        set simplified_dterm_filter_multiplier = 115
+        simplified_tuning apply
+
+        # -- Yaw lowpass --
+        set yaw_lowpass_hz = 100
+
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED): RPM Filters DShot300 (F4)
+        # -- End Defaults --
+        # -- Begin Mouse Tune --
+
+        # enable dshot rpm telemetry
+        set motor_pwm_protocol = DShot300
+        set dshot_bidir = ON
+        set motor_poles = 12
+
+        # -- Gyro lowpass filters --
+        # -- No Gyro Lowpass
+        set gyro_lpf1_static_hz = 0
+        set gyro_lpf1_dyn_min_hz = 0
+        set gyro_lpf1_dyn_max_hz = 0
+        set simplified_gyro_filter = on
+        set simplified_gyro_filter_multiplier = 150
+        simplified_tuning apply
+
+        # -- Gyro Dynamic Notches --
+        set dyn_notch_count = 1
+        set dyn_notch_q = 500
+        set dyn_notch_min_hz = 100
+        set dyn_notch_max_hz = 550
+
+        # -- RPM filtering --
+        set rpm_filter_q = 500
+        set rpm_filter_min_hz = 100
+        set rpm_filter_fade_range_hz = 60
+        set rpm_filter_harmonics = 3
+
+        # -- Dterm filtering --
+        set simplified_dterm_filter = on
+        set simplified_dterm_filter_multiplier = 115
+        simplified_tuning apply
+
+        # -- Yaw lowpass --
+        set yaw_lowpass_hz = 100
+
+    #$ OPTION END
+
+#$ OPTION_GROUP END
+
+
+#$ OPTION_GROUP BEGIN: Additional Options (Choose Many or None)
+    #$ OPTION BEGIN (CHECKED): Dynamic Idle
+        set dyn_idle_min_rpm = 30
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED): Battery Sag Compensation
+        set vbat_sag_compensation = 100
+    #$ OPTION END
+
+    #$ OPTION BEGIN (UNCHECKED): 86% Motor Limit (4s)
+         # -- Motor Output Limit for 4s --
+        set motor_output_limit = 86
```

Comment:
so this one is the only thing needed here to make 4S fly good on 3S tune?
Or how does that work?